### ISSUE TITLE
mixinLoader and cjs mixins

### ISSUE THREAD

**Issue by james2doyle** [james2doyle/ohdoylerules.com#12]
How to use common JS for mixins in Vue. Also, conditionally loading mixins.

**Comment by james2doyle** [comment]
Checker:

```js
// loads mixins based on routes
module.exports = function(name, filename) {
  // make an array no matter what
  name = (typeof(name) === 'object') ? Array.from(name) : [name];
  // are any of the names in our uri?
  const check = name.filter((n) => window.location.pathname.includes(n)).length > 0;
  return check ? require(filename) : {};
};
```

Usage:

```js
const app = new Vue({
  el: "#app",
  data: {
    // ....
  },
  mixins: [
    require('navigation'), // a regular module
    conditionalRequire('home', 'home-mixin'), // only load when "/home"
    conditionalRequire('profile', 'user-profile') // only load when "/profile"
  ]
});
```

**Comment by james2doyle** [comment]
Closed with 4c429fce2a1f4bf983227285872414cd040658df
